Add initial page related to documentation design (#217)
grafana/writers-toolkit: docs/sources/writing-guide/about-documentation-design/index.md

---
title: About our documentation design
menuTitle: Documentation design
description: Learn about Grafana's documentation docs pages

>   - 

Reviewer: ```suggestion
```

---
title: About our documentation design
menuTitle: Documentation design
description: Learn about Grafana's documentation docs pages

> aliases:

Reviewer: ```suggestion
```

>   - /docs/writers-toolkit/latest/writing-guide/about-documentation-design/

Author: ```suggestion
  - /docs/writers-toolkit/writing-guide/about-documentation-design/
```

There's no latest for Writers' Toolkit

>   - /docs/writers-toolkit/latest/writing-guide/about-documentation-design/

Reviewer 2: ```suggestion
  - 
```

weight: 100
keywords:
  - Grafana
  - documentation
  - page design
---

# About our documentation design

> In late 2022, the documentation website was significantly modified to a more modern design. The goal of the redesign was to make our technical documentation more accessible, modern, and scalable as we grow.  

Reviewer: @GrafanaWriter , Bits that include dates and words like 'new' and 'now' become stale quickly. Maybe start at "Our docs page take sadvantage of our static site generator, Hugo. As a result, several page elements are automatically managed during the publication ..." - and remove everything before that content?

If you are good with that approach, I can update the PR. Thanks!

Reviewer 2: yes, please.

> In late 2022, the documentation website was significantly modified to a more modern design. The goal of the redesign was to make our technical documentation more accessible, modern, and scalable as we grow.  

Reviewer 2: ```suggestion
The documentation website uses a more modern design approach to make our technical documentation more accessible, modern, and scalable.  
```

> In addition to a new look (including a new typeface and layout), our updated docs pages take better advantage of our static site generator, Hugo. As a result, several elements of the page are now automatically managed during the publication of the page using Hugo's taxonomy, thus the source markdown files **do not need to hand manage** these elements and **do not require** contributors to curate them.

Reviewer 2: Fixing to make more evergreen

The improvements include:

> - **Improved navigation.** The left-hand sidebar broadly outlines key topics, with nested related topics underneath. This design supports the philosophy that "every page is page one" and creates an system of documentation around a topic that is easier to reference and navigate. 

Reviewer 2: ```suggestion
- **Navigation to preview primary topics.** The left-hand sidebar broadly outlines key topics, with nested related topics underneath. This design supports the philosophy that "every page is page one" and creates an system of documentation around a topic that is easier to reference and navigate. 
```

> - **More accessible table of contents.** The table of contents now floats on the page as you scroll to the content that's hidden beneath the fold. You can also view the upcoming topics, to enable a better user experience that helps you navigate to subtopics lower on the page.

Reviewer 2: ```suggestion
- **Floating table of contents.** The table of contents floats on the page as you scroll to the content that's hidden beneath the fold. You can also view the upcoming topics, to enable a better user experience that helps you navigate to subtopics lower on the page.
```

> - **Auto-generated _Related documentation_.** Using Hugo's taxonomy, our documentation now automatically finds other documentation that's pertinent to the page you're viewing.

Reviewer 2: ```suggestion
- **Auto-generated _Related documentation_.** Using Hugo's taxonomy, our documentation automatically finds other documentation that's pertinent to the page you're viewing.
```

- **Auto-generated _Related resources from Grafana Labs_.** Hugo's taxonomy again is used to automatically generate this content.
- **Feedback.** We added more prominent options for feedback from our community.

> You can read more about the redesign in our [blog] [https://grafana.com/blog/2023/02/03/grafana-documentation-a-look-at-the-new-and-improved-design/].

Reviewer 2: ```suggestion
You can read about the redesign of our documentation pages in our [blog] [https://grafana.com/blog/2023/02/03/grafana-documentation-a-look-at-the-new-and-improved-design/].
```
